peribolos/knative-sandbox-OWNERS_ALIASES

```diff
@@ -249,6 +249,11 @@ aliases:
   - kvmware
   - psschwei
   - upodroid
+  scaling-group-approvers:
+  - dprotaso
+  - evankanderson
+  - jwcesign
+  - psschwei
```

Comment:
```suggestion
  scaling-group-approvers:
  - jwcesign
```

I think it just makes sense to add yourself


In the OWNER file for the repo you can include @psschwei and myself via the `serving-wg-leads`
https://github.com/knative/community/blob/7bd466197944e39503c107ac34503850c67cfa9a/peribolos/knative-sandbox-OWNERS_ALIASES#L268

Then add TOC as a fallback - ie see serving's [OWNERS ](https://github.com/knative/serving/blob/main/OWNERS#L3-L8 )file

Reply:
fixed